src/bosh_azure_cpi/spec/manual_tests/performance/namings/main.tf

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is the empty file required?

Reply:
it's created for the normal project structure of tf.